src/turbulence/DynamicSmagorinsky.cpp

```diff
@@ -86,8 +85,7 @@ DynamicSmagorinsky::DynamicSmagorinsky() :
     S_bar(FieldType::U),
     S_bar_f(FieldType::U),
     Cs(FieldType::U) {
-    auto params = Parameters::getInstance();
-    m_nu = params->get_real("physical_parameters/nu");
+    m_nu = settings.get_real("physical_parameters/nu");
```

Comment:
remove member variable `m_nu` instead access value through Settings

Reply:
done in 0d1d248